exp-models/addon/models/session.js

```diff
@@ -18,6 +18,9 @@ export default DS.Model.extend(AnonJamModel, {
     experimentId: DS.attr('string'),
     experimentVersion: DS.attr('string'),  // TODO: Currently this field is not acted on in any way
 
+    frameIndex: DS.attr(),
```

Comment:
Right- it was tracked internally, but that's a separate matter from deciding that every session for any experiment needs to track and save this data... 

Main areas of potential confusion include:
1. Many experiments are not surveys
2. If frameIndex is the last "page", then what is "survey page"?